src/utils/balanceHistory/getUserId.ts

```diff
@@ -0,0 +1,12 @@
+import { server$ } from "@builder.io/qwik-city";
+import jwt, { type JwtPayload } from "jsonwebtoken";
+
+export const getUserId = server$(async function () {
+  const cookie = this.cookie.get("accessToken");
+  if (!cookie) {
+    throw new Error("No cookie found");
+  }
+  const { userId } = jwt.decode(cookie.value) as JwtPayload;
+
+  return userId;
+});
```

Comment:
I'm not 100% sure but this function possibly has already been moved into separate file during previous refactoring, could you look for it? guess you could copy some lines n place them into `ctrl+shift+f` in vscode to take a quick look, if im wrong then simply resolve this comment